src/index.html

```diff
@@ -932,7 +1153,7 @@ <h2 class="controls-title">Duration</h2>
             cb(
               new Promise(function(resolve) {
                 fetch(`uv-${type}-config.json`).then(function(response) {
-                  resolve(response.json());
+                    resolve(response.json());

```

Comment:
Reverting unnecessary extra indentation:

```suggestion
                  resolve(response.json());

```